### PR TITLE
Maya/195431 link copy

### PR DESCRIPTION
**Pull request by mmmmmmaya**
### Summary
- **What:** update a bunch of links in the app and remove resources page
- **Why:** new help center ✨ 
- **Ticket:**
  - [[sc-195441]](https://app.shortcut.com/genepi/story/195441)
  - [[sc-195431]](https://app.shortcut.com/genepi/story/195431)
  - [[sc-201001]](https://app.shortcut.com/genepi/story/201001)
- **Env:** https://maya-linkcopy-frontend.dev.czgenepi.org/

### Checklist
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [x] I either asked design for a review or hid my changes behind a feature flag
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx [hunk 1]
```diff
@@ -130,8 +130,9 @@ export const CreateNSTreeModal = ({
 
   const TREE_TYPE_TOOLTIP_TEXT = (
     <div>
-      Select the Tree Type best suited for the question you are trying to anwer.{" "}
```
Comment (mmmmmmaya) [inline src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/index.tsx:133]: 😬

File: src/frontend/src/common/routes.ts [hunk 2]
```diff
@@ -1,29 +1,30 @@
 export enum ROUTES {
   HOMEPAGE = "/",
```
Comment (mmmmmmaya) [inline src/frontend/src/common/routes.ts:2]: I have always wanted to alphabetize this list

File: src/frontend/src/components/NavBar/components/RightNav/style.ts [hunk 3]
```diff
@@ -26,16 +26,6 @@ export const UploadButton = styled(Button)`
   }}
 `;
 
-export const ResourcesLink = styled.div`
```
Comment (mmmmmmaya) [inline src/frontend/src/components/NavBar/components/RightNav/style.ts:29]: unused

File: src/frontend/src/common/routes.ts (hunk 2 above)
Comment (vincent-czi) [reply src/frontend/src/common/routes.ts:2]: godspeed

**Review by vincent-czi — APPROVED** [review]

**Pull request merged**